expressEngine/customer/service/customer.js

```diff
@@ -0,0 +1,36 @@
+// import service libraries
+import { BaseService, DBService } from '../../lib/service';
+
+// import collections
+import { Customer } from '../../model';
+
+// import messages
+import { error, success } from '../../cms/customer';
+
+class Service extends BaseService {
+    registerCustomer = async (data) => {
+        try {
+            const { firstName, lastName, city, country, contactNo } = data;
+            const requiredFields = ["firstName", "lastName", "city", "country", "contactNo"];
```

Comment:
The required fields array is unused for now, the validation required function is missing, is that on purpose or did you forgot to include it?

Reply:
I have maked it comment for now.